src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenSpanBasedStringConcatTests.cs

```diff
@@ -5902,5 +6037,640 @@ .maxstack  4
               IL_003d:  ret
             }
             """);
+
+        comp = CreateCompilation(source, options: TestOptions.ReleaseExe, targetFramework: TargetFramework.Net100);
+
+        if (missingUnimportantMember.HasValue)
+        {
+            comp.MakeMemberMissing((SpecialMember)missingUnimportantMember.Value);
+        }
+
+        verifier = CompileAndVerify(comp, expectedOutput: ExecutionConditionUtil.IsCoreClr ? "abcde" : null, verify: Verification.Fails);
```

Comment:
>Fails [](http://example.com/codeflow?start=130&length=5)

FailsPEVerify?

Reply:
Similar for added tests below

Reply:
No, these all `Fail` regardless, as `InlineArrayAsReadOnlySpan` does not pass ILVerify.